src/Game/Scenes/GameSceneInputHandler.cs

```diff
@@ -921,6 +945,21 @@ internal override bool OnMouseWheel(bool up)
                 }
             }
 
+            if (CanExecuteMacro())
+            {
+                Macro macro = Macros.FindMacro(up, Keyboard.Alt, Keyboard.Ctrl, Keyboard.Shift);
+
+                if (macro != null)
+                {
+                    if (macro.Items != null && macro.Items is MacroObject mac)
```

Comment:
`if (macro.Items is MacroObject mac)`
is enough, implicitly check for nullptr

Reply:
that's what I get for copying the existing code! fixed, thanks